### PR TITLE
BulkActionPersister now handles catkeys

### LINKED ISSUE #2257: [argo/prod] NoMethodError: undefined method `[]' for nil:NilClass
It looks like this has been retrying by sidekiq for a number of days. It was originally created Thu, 17 Sep 2020 23:22:41 UTC +00:00

## Backtrace

    line 17 of [PROJECT_ROOT]/app/jobs/manage_catkey_job.rb: perform
    line 26 of [PROJECT_ROOT]/app/jobs/generic_job.rb: block in <class:GenericJob>

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/49894/faults/67839891)

```
I, [2020-09-24T05:34:48.032429 #5597]  INFO -- : [ActiveJob] [ManageCatkeyJob] [00d27436-ef19-4c55-8eec-81bae935e699] Performing ManageCatkeyJob (Job ID: 00d27436-ef19-4c55-8eec-81bae935e699) from Sidekiq(default) with arguments: 11674, {:pids=>["druid:jm950bv6322", "druid:ww757xc7815"], :manage_release=>{"tag"=>"true", "what"=>"self", "who"=>"bergeraj", "to"=>"Searchworks"}, :set_governing_apo=>{"new_apo_id"=>"druid:cw409cn2983"}, :prepare=>{"significance"=>"major", "description"=>""}, :csv_file=>nil, :groups=>["sunetid:bergeraj", "workgroup:sdr:map-staff/administrator", "workgroup:sdr:hopkins-staff/administrator", "workgroup:sdr:accessioning/administrator", "workgroup:sdr:lanciani-archive/administrator", "workgroup:sdr:spec-staff/administrator", "workgroup:sdr:administrator", "workgroup:sdr:big_local_data/administrator", "workgroup:sdr:psm-staff", "workgroup:sdr:test/administrator", "workgroup:sdr:admin-docs/administrator", "workgroup:sdr:gsb-library/administrator", "workgroup:sdr:art-staff/administrator", "workgroup:sdr:hydrus-staff/administrator", "workgroup:sdr:staff/administrator", "workgroup:sdr:metadata-staff", "workgroup:sdr:argo-message-editors/administrator", "workgroup:sdr:argo-access/administrator", "workgroup:sdr:otl", "workgroup:sdr:sohp-staff/administrator", "workgroup:sdr:admin-temporary/administrator", "workgroup:sdr:was-staff", "workgroup:sdr:eresources-staff/administrator", "workgroup:sdr:hydrus-service-team", "workgroup:sdr:catalhoyuk/administrator", "workgroup:sdr:hydrus-service-team/administrator", "workgroup:sdr:administrator-role/administrator", "workgroup:sdr:administrator-role", "workgroup:sdr:development-hourlies/administrator", "workgroup:sdr:ars-staff/administrator", "workgroup:sdr:manager-role/administrator", "workgroup:sdr:etds-registrar-staff/administrator", "workgroup:sdr:etds-quarterly-reports/administrator", "workgroup:sdr:hydrus-collection-managers", "workgroup:sdr:hydrus-staff", "workgroup:sdr:accessioning", "workgroup:sdr:manager-role", "workgroup:sdr:music-staff-temporary/administrator", "workgroup:sdr:was-staff/administrator", "workgroup:sdr:administrator/administrator", "workgroup:sdr:admin-temporary", "workgroup:sdr:developer/administrator", "workgroup:sdr:developer", "workgroup:sdr:parker-staff/administrator", "workgroup:sdr:metadata-staff-temporary/administrator", "workgroup:sdr:acquisitions-staff/administrator", "workgroup:sdr:bibliographers-staff/administrator", "workgroup:sdr:argo-message-editors", "workgroup:sdr:service-manager", "workgroup:sdr:psm-staff/administrator", "workgroup:sdr:hydrus-manager-role", "workgroup:sdr:etds-sul-staff/administrator", "workgroup:sdr:nash-photo-archive/administrator", "workgroup:sdr:serg-staff/administrator", "workgroup:sdr:curator-staff/administrator", "workgroup:sdr:law-staff/administrator", "workgroup:sdr:service-manager/administrator", "workgroup:sdr:cantor-staff/administrator", "workgroup:sdr:dpg-hourlies/administrator", "workgroup:sdr:3d-staff/administrator", "workgroup:sdr:dpg-staff/administrator", "workgroup:sdr:martin-wong/administrator", "workgroup:sdr:conservation-staff/administrator", "workgroup:sdr:hydrus-collection-managers/administrator", "workgroup:sdr:viewer-role/administrator", "workgroup:sdr:service-team/administrator", "workgroup:sdr:map-hourlies/administrator", "workgroup:sdr:crrw-project/administrator", "workgroup:sdr:sagr-staff/administrator", "workgroup:sdr:production-coordinators/administrator", "workgroup:sdr:argo-access", "workgroup:sdr:otl/administrator", "workgroup:sdr:hydrus-manager-role/administrator", "workgroup:sdr:eal-staff/administrator", "workgroup:sdr:hoover-staff/administrator", "workgroup:sdr:cidr-staff/administrator", "workgroup:sdr:sa-print-collection-staff/administrator", "workgroup:sdr:etds-sul-staff", "workgroup:sdr:lane-staff/administrator", "workgroup:sdr:role-test/administrator", "workgroup:sdr:sup-staff/administrator", "workgroup:sdr:media-microtext/administrator", "workgroup:sdr:forensics-staff/administrator", "workgroup:sdr:lockss-staff/administrator", "workgroup:sdr:smpl-staff/administrator", "workgroup:sdr:cesta-admin-staff/administrator", "workgroup:sdr:music-staff/administrator", "workgroup:sdr:metadata-staff/administrator"]}
```

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

This fixes a regression where bulk manage catkeys was broken.
Fixes #2257

### CODE REVIEW

File: app/views/bulk_actions/forms/_set_governing_apo_form.html.erb [hunk 1]
```diff
@@ -1,12 +1,6 @@
-<div>
-  <%= f.fields_for :set_governing_apo do |set_governing_apo_fields| %>
-    <div class='form-group'>
-      <div class='text'>
```
Comment (mjgiarlo) [inline app/views/bulk_actions/forms/_set_governing_apo_form.html.erb:4]: I see this text div was dropped in the changes to this file. Not needed?

**Review by mjgiarlo — APPROVED** [review]
LGTM, as long as the text div is not needed

File: app/views/bulk_actions/forms/_set_governing_apo_form.html.erb (hunk 1 above)
Comment (jcoyne) [reply app/views/bulk_actions/forms/_set_governing_apo_form.html.erb:4]: Nope, it wasn't doing anything useful.  It didn't have a style.

Comment (mjgiarlo) [reply app/views/bulk_actions/forms/_set_governing_apo_form.html.erb:4]: @jcoyne ok, great!

**Pull request merged**